.github/workflows/account-deploy.yml

```diff
@@ -7,6 +7,11 @@ on:
         description: "The terraform workspace to target for account actions"
         required: true
         type: string
+      plan_only:
+        description: "Whether to only plan the deployment, not run it"
+        required: false
+        type: boolean
+        default: false
```

Comment:
Maybe the default of the `plan_only` be false so that if somebody missed setting it on a new workflow it didn't potentially mess things up by default

Reply:
👍🏽 Switched to an `apply` variable so that it's a safer default and explicitly opt-in